microsetta_public_api/_io.py

```diff
@@ -15,4 +56,8 @@ def _dict_of_paths_to_beta_data(dict_of_qza_paths, resource_name) -> \
                                            DistanceMatrixType,
                                            view_type=DistanceMatrix,
                                            )
+    key = list(new_resource.keys())[0]
+    dm = new_resource[key]
+    k = min(len(dm.ids) - 1, 100)
```

Comment:
...this is hardcoded at a max of 100 neighbors. It *may* be nice to push to config, but I vote another problem for another day.